source/_integrations/comelit.markdown

```diff
@@ -23,5 +24,10 @@ The Comelit SimpleHome integration allows you to control your [Comelit home auto
 There is support for the following platform types within Home Assistant:
 
 - **Comelit Serial Bridge** - allows local control for light, cover, irrigation and "other" devices; creates power sensors.
+- **Comelit VEDO System** - allows local control of the alarm system.
 
 {% include integrations/config_flow.md %}
+
+## Alarm control panel
+
+The integration will create an alarm entity for each {% term area %} and a sensor for each {% term zone %}.
```

Comment:
Adjusted this one, as these are zones from the alarm system, not Home Assistant (which the terms refer to).

```suggestion
## Alarm control panel

The integration will create an alarm entity for each area and a sensor for each zone.
```